src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -71,6 +71,33 @@ AttributRadiationFlag
 
 /*########################### define species #################################*/
 
+/*--------------------------- photons -------------------------------------------*/
+
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+
+value_identifier(float_X, DummyMass, 1.0);
+value_identifier(float_X, DummyCharge, 0.0);
```

Comment:
OK, then we have to postpone removing these dummy values. They are fine for now. :+1: